runtimes/php/php-apps.md

```diff
@@ -6,6 +6,7 @@ shortdesc: PHP is a widely-used general-purpose scripting language that is espec
 # Deploy PHP apps
 
 PHP is available on our platform with the branches 5.4 and 5.5. You can use FTP or Git to deploy your applications.
+As HTTP Server, we use Apache 2.
```

Comment:
The HTTP server is apache 2, and the PHP code is executed by PHP-FPM (with a link on apache2 and php-fpm)